### PR TITLE
Austenem/Workspaces hackathon fixes (CAT-906, CAT-909, CAT-910, & CAT-911) 

### PR DESCRIPTION
**Pull request by austenem**
## Summary

[CAT-906](https://hms-dbmi.atlassian.net/browse/CAT-906?atlOrigin=eyJpIjoiYWZhMDA0YzYwMzgwNDlmMTkzZmNkMjQxM2FiYTIzYzQiLCJwIjoiaiJ9): Broken "Copy HuBMAP IDs button"

[CAT-909](https://hms-dbmi.atlassian.net/browse/CAT-909?atlOrigin=eyJpIjoiMTRjZDQwMTI3MTM3NGQzM2JhZmEyZWU0ZjJiNmY3OWUiLCJwIjoiaiJ9): Too many marks on Advanced Config slider

**Before:**

screenshot_2024-09-19_at_10 37 30_am_720

**After:**

Screenshot 2024-09-19 at 3 16 00 PM

[CAT-910](https://hms-dbmi.atlassian.net/browse/CAT-910?atlOrigin=eyJpIjoiNDNkY2JjMzIyNGRhNDEyNWI3ODViNjhmMWVkYTM5YTMiLCJwIjoiaiJ9): Unnecessary tooltip on template grid

**Before:**

Screenshot 2024-09-19 at 12 21 37 PM

**After:**

Screenshot 2024-09-19 at 3 16 54 PM

[CAT-911](https://hms-dbmi.atlassian.net/browse/CAT-911?atlOrigin=eyJpIjoiN2Q5NWNmZjM3YzczNDMyN2E0ZDI5OGZiMTEwMjA3MWQiLCJwIjoiaiJ9): Incorrect job type for template example

**Before:**

Screenshot 2024-09-19 at 3 17 32 PM

**After:**

Screenshot 2024-09-19 at 3 17 58 PM

## Checklist

- [X] Code follows the project's coding standards
  - [X] Lint checks pass locally
  - [X] New `CHANGELOG-your-feature-name-here.md` is present in the root directory, describing the change(s) in full sentences.
- [ ] Unit tests covering the new feature have been added
- [X] All existing tests pass
- [X] Any relevant documentation in JIRA/Confluence has been updated to reflect the new feature
- [X] Any new functionalities have appropriate analytics functionalities added

[CAT-906]: https://hms-dbmi.atlassian.net/browse/CAT-906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CAT-909]: https://hms-dbmi.atlassian.net/browse/CAT-909?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CAT-910]: https://hms-dbmi.atlassian.net/browse/CAT-910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CAT-911]: https://hms-dbmi.atlassian.net/browse/CAT-911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: context/app/static/js/hooks/useCopyText.ts [hunk 1]
```diff
@@ -6,17 +6,19 @@ export const useHandleCopyClick = () => {
...
-        .then(() => {
-          toastSuccess(`Copied to clipboard.${additionalText ? ` ${additionalText}` : ''}`);
-        })
-        .catch((e) => {
-          console.error('Error copying to clipboard', e);
-          toastError(
-            `Encountered an error while copying, please try again.${additionalText ? ` ${additionalText}` : ''}`,
-          );
-        });
+      setTimeout(() => {
```
Comment (NickAkhmetov) [inline context/app/static/js/hooks/useCopyText.ts:9]: We should document the necessity of this `setTimeout`, e.g. how it fixes the issue at hand.

**Comment by john-conroy** [comment]
Great, each of these bugs seem to be resolved when checking out locally.

**Review by NickAkhmetov — APPROVED** [review]
I'm a bit curious about how the `copy to clipboard` workaround functions, but otherwise have no blocking concerns.

**Review by john-conroy — APPROVED** [review]
Let's get this in.

File: context/app/static/js/hooks/useCopyText.ts (hunk 1 above)
Comment (austenem) [reply context/app/static/js/hooks/useCopyText.ts:9]: Agreed - not entirely sure why this fix works, I followed [this thread](https://stackoverflow.com/questions/62327358/javascript-clipboard-api-safari-ios-notallowederror-message) about the console error that was provided. Looks like wrapping the call in a `setTimeout` re-contextualizes this action as a user event?

**Pull request merged**